controllers/serverTableController.js

```diff
@@ -1,27 +1,65 @@
-var app = angular.module('pokemonDraft', []);
-app.controller('serverTableController', function($scope, $http) {
+var app = angular.module('pokemonDraft', ['ui.bootstrap.modal', 'ui.bootstrap.tpls']);
```

Comment:
in terms of that var I think it is still valid as it is global space,  one "app" declaration to many controllers